tests/unit/test_signers.py

```diff
@@ -727,7 +728,7 @@ def test_generate_presigned_url(self):
             'query_string': {},
             'url_path': u'/mybucket/mykey',
             'method': u'GET',
-            'context': {}}
+            'context': mock.ANY}
```

Comment:
This was switched to `mock.ANY` because we are now emitting the client parameter related events that inject values into the context. Thus using the context's exact value for these tests will be a maintenance burden if anymore customizations are added that inject into the context.

Reply:
I don't think review comments on your own PR are the best place for these type of contextual comments.  Either a) A comment in the code or b) in the comment message.